api/openapi/api.yaml

```diff
@@ -303,6 +303,7 @@ paths:
           schema:
             type: string
             enum: ["replace", "upsert"]
+          required: true
```

Comment:
If not setting this as "required", users could pass in None for "table_manipulation" which could end up breaking the downstream function.